src/config/app.config.module.ts

```diff
@@ -22,6 +25,10 @@ import { AuthModule } from 'src/modules/auth/auth.module';
         path: 'auth',
         module: AuthModule,
       },
+      {
+        path: 'fee-challan',
```

Comment:
I think we can also use this function I created for auth. It's really handy and keeps your paths very readable.
[Link](https://github.com/CyFy-Blockchain/auth/blob/master/src/modules/modules.module.ts#L8-L27) to the function

Reply:
Noted, thanks